config/routes.rb

```diff
@@ -1,21 +1,20 @@
 Rails.application.routes.draw do
-  root 'contacts#call_list'
+  root 'needs#index'
 
   resources :contacts, only: [:index, :show, :edit, :update, :new] do
     resources :needs, only: [:new, :create]
-    collection do
-      get 'call-list'
-    end
+    # collection do
+    #   get 'call-list'
+    # end
     get 'triage', to: 'triage#edit', as: 'edit_triage'
     put 'triage', to: 'triage#update', as: 'triage'
   end
-  get '/contacts/:id/needs', to: 'contacts#needs'
+  # get '/contacts/:id/needs', to: 'contacts#needs'
 
   resources :needs, only: [:index, :show, :edit, :update] do
```

Comment:
Just had a thought: maybe we should make viewing/editing a need always part of the contacts resource? So you can go to `/needs`, but when you click one, instead of going to `/needs/123`, you're always taken to `/contacts/23/needs/123`. Maybe that's crazy though. Just thinking out loud.

Reply:
There are effectively two views required - one is the collection of people with needs, and the other is the collection of needs across everyone.  so it makes sense to have the `/needs` endpoint.